.gitignore

```diff
@@ -35,6 +35,7 @@ venv/
 yarn-error.log
 oppia-release-credentials.json
 sheets.googleapis.com-python.json
+.env
```

Comment:
Add `.env` so I can use the pylint extension on vscode. Otherwise, the python program has no idea where to find the packages.